### PR TITLE
Added bannerURL/splashURL typings

### PR DESCRIPTION
**Pull request by oathompsonjones**

### CODE REVIEW

**Review by reboxer — COMMENTED** [review]
maybe also add splashURL?

File: index.d.ts [hunk 1]
```diff
@@ -1225,7 +1225,9 @@ declare namespace Eris {
     joinedAt: number;
     ownerID: string;
     splash?: string;
+    splashURL?: string;
```
Comment (Skillz4Killz) [inline index.d.ts]:
```suggestion
    splashURL: string | null;
```

File: index.d.ts [hunk 2]
```diff
@@ -1225,7 +1225,9 @@ declare namespace Eris {
...
     banner?: string;
+    bannerURL?: string;
```
Comment (Skillz4Killz) [inline index.d.ts]:
```suggestion
    bannerURL: string | null;
```

**Review by Skillz4Killz — COMMENTED** [review]
Using `?` to mark something as optional makes typescript understand it as if it is `undefined`. The actual behavior in Eris never allows this to be undefined. It marks it as `null` making this optional and not explicitly null can cause issues to end users.

**Comment by oathompsonjones** [comment]
Wasn't aware those would return null as opposed to undefined, thanks for letting me know.

**Review by Skillz4Killz — APPROVED** [review]

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**